packages/0x.js/src/fetchers/simple_balance_and_proxy_allowance_fetcher.ts

```diff
@@ -0,0 +1,26 @@
+import { BlockParamLiteral } from '@0xproject/types';
+import { BigNumber } from '@0xproject/utils';
+
+import {BalanceAndProxyAllowanceFetcher} from '../abstract/balance_and_proxy_allowance_fetcher';
+import {TokenWrapper} from '../contract_wrappers/token_wrapper';
+
+export class SimpleBalanceAndProxyAllowanceFetcher implements BalanceAndProxyAllowanceFetcher {
+    private _token: TokenWrapper;
```

Comment:
Let's name this `_tokenWrapper`. We only call it `token` in 0x.ts because we can the public interface to be `zeroEx.token.getBalance()`, which is nicer then `zeroEx.tokenWrapper...`.